community/docker/APKBUILD

```diff
@@ -36,7 +37,7 @@ source="
 	runc-$_runc_ver.tar.gz::https://github.com/opencontainers/runc/archive/$_runc_ver.tar.gz
 	containerd-$_containerd_ver.tar.gz::https://github.com/docker/containerd/archive/$_containerd_ver.tar.gz
 	libnetwork-$_libnetwork_ver.tar.gz::https://github.com/docker/libnetwork/archive/$_libnetwork_ver.tar.gz
-
+	tini-$_tini_ver.tar.gz::https://github.com/krallin/tini/archive/$_tini_ver.tar.gz
```

Comment:
@ncopa I have added a shell wrapper for this in my local copy (didn't push it yet). Also fixed the git dependency (actually needs to be a make dependency). Also re-enabled aufs (any idea why it's off?). Now if you look the PR is failing because it can't find a header file when trying compile with btrfs-progs-dev (not sure what to do here. I could also disable btrfs graphdriver)